snuba/datasets/schemas/tables.py

```diff
@@ -47,13 +53,15 @@ def __init__(
         storage_set_key: StorageSetKey,
         mandatory_conditions: Optional[Sequence[FunctionCall]] = None,
         part_format: Optional[Sequence[util.PartSegment]] = None,
+        write_format: WriteFormat = WriteFormat.JSON
     ):
         self.__local_table_name = local_table_name
         self.__dist_table_name = dist_table_name
         self.__storage_set_key = storage_set_key
         self.__columns = columns
         self.__mandatory_conditions = mandatory_conditions
         self.__part_format = part_format
+        self.__write_format = write_format
```

Comment:
The TableSchema is meant to provide the table config for both readable and writable storages. The write format is only relevant for the WritableStorage. Was there a way to fit this in the WritableStorage class ?